### ISSUE TITLE
Crashing after unpredictable amount of time

### ISSUE THREAD

**Issue by rozza-m** [circa10a/cv-notifier#3]
I'm getting this error and cv-notifier crashes out:

```
Traceback (most recent call last):
  File "/root/cv-notifier/cv-notifier/./main.py", line 74, in <module>
    main()
  File "/root/cv-notifier/cv-notifier/./main.py", line 66, in main
    requests.process(webhooks_to_process, detected_object_name, detected_object_confidence)
  File "/root/cv-notifier/cv-notifier/lib/webhooks/requests.py", line 64, in process
    if seconds_since_last_notified < webhook['notifyInterval']:
KeyError: 'notifyInterval'
FATAL: exception not rethrown
Aborted
```

There doesn't seem to be a problem with just restarting, but better to fix it at source I guess :)

**Comment by rozza-m** [comment]
Not quite sure why 'notifyInterval' doesn't exist sometimes, but perhaps it can be solved by substituting with a default if it's not found

```
notify_interval = webhook.get('notifyInterval', 900)
if seconds_since_last_notified < notify_interval:
```

**Comment by circa10a** [comment]
Oh jeez 🤦 
I introduced this bug with the last fix, so sorry! This has been resolved in `v0.1.8`. I validated all is well on my local setup :)

Thank you again for reporting!

**Comment by rozza-m** [comment]
Thanks, I've updated but of course it's hard to immediately verify the lack of crashing, so I suggest close this and I'll re-open if I see the issue again. Thank you!
